mixins/test/localize-mixin.html

```diff
@@ -10,7 +10,7 @@
 		<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
 		<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
 		<script type="module" src="../demo/localize-test.js"></script>
-		<script type="module" src="../demo/localize-test-no-resources.js"></script>
```

Comment:
This component/fixture weren't actually used, and don't really make sense now that the mixin ONLY does localization.